src/load_order.ts

```diff
@@ -22,14 +56,39 @@ import {
   VortexWrappedSerializeFunc,
   VortexWrappedValidateFunc,
 } from "./vortex-wrapper";
+import { REDMOD_INFO_FILENAME } from "./installers.layouts";
+import { jsonp } from "./installers.utils";
+import { fileFromDiskTE } from "./installers.shared";
+import {
+  ModAttributeName,
+  ModType,
+} from "./installers.types";
 
 // Ensure we're using win32 conventions
 const path = win32;
 
+
+// Defs
+
+const LOAD_ORDER_FILE_NAME =
+  `${EXTENSION_NAME_INTERNAL}-redmod-load-order.json`;
+
+const LOAD_ORDER_FILE_RELATIVE_PATH =
+  path.join(`r6\\cache\\modded\\${LOAD_ORDER_FILE_NAME}`);
```

Comment:
I don't think we need this. This seems to me like we are gonna use our own file, but I think it is better to use the file that already exists from the redmod. This simplifies things and won't cause issues with the UI and file being incongruous.

Reply:
I mean we should be using `const modListPath = path.join(discoveryPath, 'r6', 'cache', 'modded', 'mods.json');` for the load order file, and deserialize and serialize to it.

Reply:
We have to run `redmod deploy`, which generates that file. It is needed for script compilation plus it’s possible it does validation that we don’t want to reproduce.